### PR TITLE
CHSDI bfe.thermische.netze

### PR DESCRIPTION
**Pull request by Simon1193**
[testlink](https://mf-chsdi3.int.bgdi.ch/ltbei_thermische_netze/shorten/85b97d1635)

### CODE REVIEW

**Comment by ltclm** [comment]
do you have a testlink?

**Comment by Simon1193** [comment]
> do you have a testlink?

**Comment by Simon1193** [comment]
testlink now available

File: chsdi/templates/htmlpopup/thermischenetze.mako [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      <td class="cell-meta">${_('ch.bfe.thermische-netze.operatoraddress')}</td>
+      <td>${c['attributes']['operatoraddress'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-meta">${_('ch.bfe.thermische-netze.phone')}</td>
+      <td>${c['attributes']['phone'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-meta">${_('ch.bfe.thermische-netze.web')}</td>
+      <td>${c['attributes']['web'] or '-'}</td>
```
Comment (AFoletti) [inline chsdi/templates/htmlpopup/thermischenetze.mako]: Please make this a clickable link

File: chsdi/templates/htmlpopup/thermischenetze.mako [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    </tr>
+    <tr>
+      <td class="cell-meta">${_('ch.bfe.thermische-netze.mail')}</td>
+      <td>${c['attributes']['mail'] or '-'}</td>
```
Comment (AFoletti) [inline chsdi/templates/htmlpopup/thermischenetze.mako]: Here too, please make it a mailto:

File: chsdi/templates/htmlpopup/thermischenetze.mako [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      <td class="cell-meta">${_('ch.bfe.thermische-netze.power')}</td>
+      <td>${c['attributes']['power'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-meta">${_('ch.bfe.thermische-netze.energy')}</td>
+      <td>${c['attributes']['energy'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-meta">${_('ch.bfe.thermische-netze.houseconnections')}</td>
+      <td>${c['attributes']['houseconnections'] or '-'}</td>
```
Comment (AFoletti) [inline chsdi/templates/htmlpopup/thermischenetze.mako]: I guess this is an INT. Please show it without decimals

**Comment by AFoletti** [comment]
Ah, another thing. I always get double tooltip. Did you perchance forgot to truncate the table last time you tested the import?

**Comment by Simon1193** [comment]
> Ah, another thing. I always get double tooltip. Did you perchance forgot to truncate the table last time you tested the import?

Problem fixed. I had two similar classes on model.py (one is a view) and I forgot to supress one

**Comment by AFoletti** [comment]
LGTM. I'll send the testlink to the customer

**Comment by Simon1193** [comment]
> Another small issue.
> In the extended tooltip, the "web" and "mail" attributes are shown as link to "none" when they are empty in the database. See [here](https://mf-chsdi3.int.bgdi.ch/ltbei_thermische_netze/rest/services/inspire/MapServer/ch.bfe.thermische-netze/242/extendedHtmlPopup?lang=de)
> When there is no value for those attributes, a "-" should be shown instead.
> 
> Thanks!

Changes made. Instead of "-", I replace missing value by a empty space (cause otherwise it considered "-" as an url link

**Pull request merged**